src/renderer_gl.h

```diff
@@ -1375,7 +1375,7 @@ namespace bgfx { namespace gl
 			bx::memCopy(m_unboundUsedAttrib, m_used, sizeof(m_unboundUsedAttrib) );
 		}
 
-		void bindAttributes(const VertexLayout& _vertexDecl, uint32_t _baseVertex = 0);
+		void bindAttributes(const VertexLayout& layout, uint32_t _baseVertex = 0);
```

Comment:
Use underscore on arguments... Like `_layout`

Reply:
I'll fix that

Reply:
Check other instance too, I just noticed this one, but there might be more.

Reply:
That was the only occurrence.